citesphere/src/main/webapp/WEB-INF/views/auth/authorities/create.html

```diff
@@ -352,6 +353,10 @@ <h4 class="modal-title" id="authorLabel">Authority Search Result</h4>
                                 </tbody>
                             </table>
                             
+                            <div class="loader" style="font-size: 30px; text-align: center;">
+                				<i id="searchAuthorPageSpinner" class="icon-spinner" style="color: blue"></i>
+            				</div>
```

Comment:
Indentation needs a little correction. Note that indentation shown in eclipse is not correct even though it displays as such.

Reply:
Fixed this